packages/contracts-bedrock/test/CommonTest.t.sol

```diff
@@ -31,46 +37,186 @@ import { AddressManager } from "src/legacy/AddressManager.sol";
 import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
 import { LegacyMintableERC20 } from "src/legacy/LegacyMintableERC20.sol";
+import { LegacyMessagePasser } from "src/legacy/LegacyMessagePasser.sol";
 import { SystemConfig } from "src/L1/SystemConfig.sol";
 import { ResourceMetering } from "src/L1/ResourceMetering.sol";
 import { Constants } from "src/libraries/Constants.sol";
+import { Deploy } from "scripts/Deploy.s.sol";
 
-contract CommonTest is Test {
+contract CommonTest is Deploy, Test {
     address alice = address(128);
     address bob = address(256);
-    address multisig = address(512);
 
-    address immutable ZERO_ADDRESS = address(0);
-    address immutable NON_ZERO_ADDRESS = address(1);
-    uint256 immutable NON_ZERO_VALUE = 100;
-    uint256 immutable ZERO_VALUE = 0;
-    uint64 immutable NON_ZERO_GASLIMIT = 50000;
-    bytes32 nonZeroHash = keccak256(abi.encode("NON_ZERO"));
-    bytes NON_ZERO_DATA = hex"0000111122223333444455556666777788889999aaaabbbbccccddddeeeeffff0000";
+    bytes32 constant nonZeroHash = keccak256(abi.encode("NON_ZERO"));
 
     event TransactionDeposited(address indexed from, address indexed to, uint256 indexed version, bytes opaqueData);
 
     /// @dev OpenZeppelin Ownable.sol transferOwnership event
     event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
 
+    OptimismPortal optimismPortal;
+    L2OutputOracle l2OutputOracle;
+    SystemConfig systemConfig;
+    L1StandardBridge l1StandardBridge;
+    L1CrossDomainMessenger l1CrossDomainMessenger;
+    AddressManager addressManager;
+    L1ERC721Bridge l1ERC721Bridge;
+    OptimismMintableERC20Factory l1OptimismMintableERC20Factory;
+    ProtocolVersions protocolVersions;
+
+    L2CrossDomainMessenger l2CrossDomainMessenger;
+    L2StandardBridge l2StandardBridge;
+    L2ToL1MessagePasser l2ToL1MessagePasser;
+    OptimismMintableERC20Factory l2OptimismMintableERC20Factory;
+    L2ERC721Bridge l2ERC721Bridge;
+    BaseFeeVault baseFeeVault;
+    SequencerFeeVault sequencerFeeVault;
+    L1FeeVault l1FeeVault;
+    GasPriceOracle gasPriceOracle;
+    L1Block l1Block;
+    LegacyMessagePasser legacyMessagePasser;
+    GovernanceToken governanceToken;
+
     FFIInterface ffi;
 
-    function setUp() public virtual {
+    function setUp() public virtual override {
         // Give alice and bob some ETH
         vm.deal(alice, 1 << 16);
         vm.deal(bob, 1 << 16);
-        vm.deal(multisig, 1 << 16);
 
         vm.label(alice, "alice");
         vm.label(bob, "bob");
-        vm.label(multisig, "multisig");
 
         // Make sure we have a non-zero base fee
         vm.fee(1000000000);
 
+        // Set the deterministic deployer in state
+        vm.etch(
+            0x4e59b44847b379578588920cA78FbF26c0B4956C,
+            hex"7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3"
+        );
```

Comment:
I believe this should already be deployed by default
```suggestion
```

Reply:
I was weirdly getting an error that it did not exist and doing this fixed that error

Reply:
Could be a forge version thing, not a big deal either way so will resolve this